### ISSUE TITLE
Slack integrations

### ISSUE THREAD

**Issue by durden** [pluralsight/guides-cms#99]
It would be cool to have some integration with [slack](https://slack.com).  Just a few random ideas:
- Bot to praise a user in our [slack community](https://hackguides.slack.com/messages/general/) when a pull request is merged
- Post in slack channel each time a new guide is moved to in-review or published

**Comment by MicroTransactionsMatterToo** [comment]
This should be easy using GitHub webhooks
 Haven't done anything with bot's, but the Bitbucket one just uses an API token and lets you set your own triggers

**Comment by durden** [comment]
Good idea.  We're already using github web hooks for [several things](https://github.com/pluralsight/guides-cms/blob/master/pskb_website/webhooks.py) including cache invalidation.  So, it shouldn't be too hard to use the existing `push_event` endpoint to handle something like this.
